test/cpp/jit/test_gpu.cpp

```diff
@@ -1116,28 +1116,24 @@ void testGPU_FusionParser() {
   // 1. this can be moved to a dedicated "golden" file
   // 2. use a fuzzy compare (ignore non-significant whitespaces for example)
   const std::string expected_kernel = R"(
-__global__ void CUDAGeneratedKernel(Tensor<float, 1> T0, Tensor<float, 1> T1, Tensor<float, 1> T3){
+__global__ void CUDAGeneratedKernel(Tensor<float, 1> T0, Tensor<float, 1> T1, Tensor<float, 1> T3) {
   float T2[1];
-  if ( ( ( ( ( ( blockIdx.x * 1 ) + ( 1 - 1 ) ) * 128 ) + threadIdx.x ) < T0.size[0] ) ) {
-    for(size_t i6 = 0; i6 < 1; ++i6 ) {
-      T2[ i6 ]
-         = T0[ ( ( ( ( blockIdx.x * 1 ) + i6 ) * 128 ) + threadIdx.x ) ]
-         * T1[ ( ( ( ( blockIdx.x * 1 ) + i6 ) * 128 ) + threadIdx.x ) ];
-      T3[ ( ( ( ( blockIdx.x * 1 ) + i6 ) * 128 ) + threadIdx.x ) ]
-         = T2[ i6 ]
-         * T0[ ( ( ( ( blockIdx.x * 1 ) + i6 ) * 128 ) + threadIdx.x ) ];
+  if ((((((blockIdx.x * 1) + (1 - 1)) * 128) + threadIdx.x) < T0.size[0])) {
+    for(size_t i6 = 0; i6 < 1; ++i6) {
+      T2[i6]
+         = T0[((((blockIdx.x * 1) + i6) * 128) + threadIdx.x)] * T1[((((blockIdx.x * 1) + i6) * 128) + threadIdx.x)];
+      T3[((((blockIdx.x * 1) + i6) * 128) + threadIdx.x)]
+         = T2[i6] * T0[((((blockIdx.x * 1) + i6) * 128) + threadIdx.x)];
```

Comment:
I like collapsing the parenthesis, but I'd prefer to have the operators on separate lines for readability.
```
T2[]
 = T0[]
 *  T1[]
```
For reads and writes, I'm fine with this: `T2[] = T0[]`

Reply:
I agree. The formatting is not final, and I was planning to revisit it in a follow up PR to keep the changes a bit smaller (and also since we have opportunities to improve the formatting while also simplify the codegen code itself)

But if this is something we care strongly, I'd be happy to update the PR.

Reply:
I'm just adding my 2 cents on the kernel formatting. :smile: 

I also noticed that the for-loop is redundant, since it is only run once.

Reply:
> I also noticed that the for-loop is redundant, since it is only run once.

Yep. That's a completely different beast altogether though. We're not doing any low-level optimizations today (but we could, and probably should - another reason to have a standalone kernel IR)